google/detectors/credentials/generic_weak_credential_detector/src/main/java/com/google/tsunami/plugins/detectors/credentials/genericweakcredentialdetector/testers/mlflow/MlFlowCredentialTester.java

```diff
@@ -85,8 +85,9 @@ public boolean canAccept(NetworkService networkService) {
                 .bodyString()
                 .get()
                 .contains(
-                    "You are not authenticated. "
-                        + "Please see https://www.mlflow.org/docs/latest/auth/index.html#authenticating-to-mlflow "
+                    "You are not authenticated. Please see "
+                        + "https://www.mlflow.org/docs/latest/auth/index.html"
+                        + "#authenticating-to-mlflow"
```

Comment:
Missing space at the end of this string, causes `detect_canAccept` test to fail.